modules/ballerina-core/src/main/java/org/ballerinalang/services/dispatchers/jms/JMSServiceDispatcher.java

```diff
@@ -46,15 +47,16 @@
     private Map<String, Service> serviceMap = new HashMap<>();
 
     @Override
-    public Service findService(CarbonMessage cMsg, CarbonCallback callback, Context balContext) {
+    public Service findService(CarbonMessage cMsg, CarbonCallback callback, Context balContext)
+            throws ServiceNotFoundException {
         Object serviceIdProperty = cMsg.getProperty(Constants.JMS_SERVICE_ID);
         String serviceId = (serviceIdProperty != null) ? serviceIdProperty.toString() : null;
         if (serviceId == null) {
-            throw new BallerinaException("Service Id is not found in JMS Message", balContext);
+            throw new ServiceNotFoundException("Service Id is not found in JMS Message", balContext);
```

Comment:
As I see in ServerConnectorMessageHandler.java:71 "service" has been written in lowercase. Shouldn't we follow the same here for "Message"?